src/test/java/org/folio/rest/impl/RtacResourceImplTest.java

```diff
@@ -396,7 +396,7 @@ public final void testGetRtacByIdVolumeFormatting() {
     final JsonObject expectedJson = new JsonObject(
```

Comment:
This code wasn't touched, but if possible, the "json" object declared a line above (395) could be renamed to "respJson" or "responseJson" to make the name more meaningful.